src/TTextEdit.cpp

```diff
@@ -1043,20 +1067,12 @@ void TTextEdit::mousePressEvent(QMouseEvent* event)
         if (event->modifiers() & Qt::ControlModifier) {
             mCtrlSelecting = true;
         }
-        int x = event->x() / mFontWidth;
-        if (mShowTimeStamps) {
-            if (x < 13) {
-                mCtrlSelecting = true;
```

Comment:
Removing this line of code(and the conditional path to it) is presumably what broke timestamp selection. Probably could be re-implemented pretty easily.